crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -125,5 +137,11 @@ def addon_installed?(addon)
     rescue NameError
       false
     end
+
+    def repo_version_available(product, version)
+      `zypper products`.split("\n").detect do |p|
```

Comment:
> This will pick only the first ocurence of product. 
> So I'd say, just put the version check inside as well.

good catch, thanks

> Good test case for your unit tests...

seems so, but unfortunately the tests can't really use zypper, as it may not be available (travis), and the repos wouldn't exist. so I need to skip (mock) it...